### PR TITLE
full stack

### PR DESCRIPTION
**Pull request by AWAlemu**

### CODE REVIEW

**Review by bryantee — COMMENTED** [review]
Nice work! I digg it.

File: readme [hunk 1]
```diff
@@ -0,0 +1,9 @@
+This is a simple shopping list app that provides the following:
+
+1, Creating a user account
+2, Adding an item to the list
+3, Editing esisting items
```
Comment (navkelley) [inline readme]: Line 5 of the readme.md has a typo/misspelled word

File: public/app.js [hunk 2]
```diff
@@ -43,7 +43,7 @@ $(document).ready(function() {
 	authenticate();
     var userInput = $('input[name=item]');
 	var addButton = $('#add-to-list');
```
Comment (navkelley) [inline public/app.js]: Not sure if this extra indentation is intentional

**Review by navkelley — COMMENTED** [review]
Just a few things I saw. Learning how to do my first code review. Thanks for posting!

File: public/app.js (hunk 2 above)
Comment (AWAlemu) [reply public/app.js]: Thanks for the feedback. I've made corrections per your suggestions.

**Comment by lsroufe** [comment]
Hello I'm also doing my first code review

**Pull request merged**